.circleci/config.yml

```diff
@@ -43,9 +43,6 @@ jobs:
             "$CIRCLE_BUILD_URL" > version.json
           # yamllint enable rule:line-length
 
-      - store_artifacts:
-          path: version.json
```

Comment:
I don't know why we ever did this. We don't use this artifact anywhere as near as I can tell.